README.md

```diff
@@ -1,34 +1,55 @@
-# Membrane Multimedia Framework: Template Plugin
+# Membrane FFmpeg SWScale plugin
 
-[![Hex.pm](https://img.shields.io/hexpm/v/membrane_template_plugin.svg)](https://hex.pm/packages/membrane_template_plugin)
-[![CircleCI](https://circleci.com/gh/membraneframework/membrane_template_plugin.svg?style=svg)](https://circleci.com/gh/membraneframework/membrane_template_plugin)
+[![Hex.pm](https://img.shields.io/hexpm/v/membrane_ffmpeg_swscale_plugin.svg)](https://hex.pm/packages/membrane_ffmpeg_swscale_plugin)
+[![API Docs](https://img.shields.io/badge/api-docs-yellow.svg?style=flat)](https://hexdocs.pm/membrane_ffmpeg_swscale_plugin/)
+[![CircleCI](https://circleci.com/gh/membraneframework/membrane_ffmpeg_swscale_plugin.svg?style=svg)](https://circleci.com/gh/membraneframework/membrane_ffmpeg_swscale_plugin)
 
-This repository contains a template for new plugins.
+Plugin performing video scaling, using SWScale module of [FFmpeg](https://www.ffmpeg.org/) library.
 
-Check out different branches for other flavours of template.
-
-It is part of [Membrane Multimedia Framework](https://membraneframework.org).
+It is a part of [Membrane Multimedia Framework](https://membraneframework.org).
 
 ## Installation
 
-The package can be installed by adding `membrane_template_plugin` to your list of dependencies in `mix.exs`:
+Add the following line to your `deps` in `mix.exs`. Run `mix deps.get`.
 
 ```elixir
-def deps do
-  [
-    {:membrane_template_plugin, "~> 0.1.0"}
-  ]
-end
+{:membrane_ffmpeg_swscale_plugin, "~> 0.1.0"}
 ```
 
-## Usage
+You also need to have [FFmpeg](https://www.ffmpeg.org/) library installed.
+
+## Sample usage
 
-TODO
+```elixir
+defmodule Scaling.Pipeline do
+  use Membrane.Pipeline
+
+  @doc false
+  @impl true
+  def handle_init(_) do
+    children = [
+      file_src: %Membrane.File.Source{location: "/tmp/input.raw"},
```

Comment:
To make it clear for the user, explain above what the file should contain and what will be the result. You can also make it a bit more practical by adding decoder and encoder to the pipeline and explain that the file should contain encoded video